packages/typespec-go/test/cadlranch/parameters/bodyoptionalgroup/zz_options.go

```diff
@@ -20,10 +20,16 @@ type BodyOptionalityClientRequiredImplicitOptions struct {
 // method.
 type BodyOptionalityOptionalExplicitClientOmitOptions struct {
 	Body *BodyModel
+
+	// Body parameter's content type. Known values are application/json
+	ContentType *string
```

Comment:
This is due to https://github.com/Azure/autorest.go/issues/1444 and will go away once it's fixed.